addons/common/functions/fnc_switchAttachmentMode.sqf

```diff
@@ -49,13 +55,15 @@ switch (currentWeapon _unit) do {
             ["CBA_attachmentSwitched", _this] call CBA_fnc_localEvent;
         }, [_unit, _currItem, _switchItem, _currWeaponType]] call CBA_fnc_execNextFrame;
     };
+    default { ERROR_1("bad weapon - %1",_this) };
```

Comment:
We should do an early exist after the switch case, so that we don't display any sort of notifications.